networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -26,7 +26,12 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 [id="{context}_configuration-additional-network-attachment"]
 == Configuration for an additional network attachment
 
-An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group. The configuration for the API is described in the following table:
+An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group.
+[IMPORTANT]
+====
+Do not store any sensitive information or a secret in the `NetworkAttachmentDefinition` object because this information is accessible by the unprivileged user.  
```

Comment:
I don't know that we'd call this an 'unprivileged' user. From the BZ, it looks like access to this information was expanded from only cluster admin to both cluster and project admins. But there are other roles that wouldn't have access.

Reply:
it should be project admin user